src/parameters.ml

```diff
@@ -8,8 +8,8 @@ open Constants
 
 type parameters =
   { q : fixedpoint; (* 1/kit, really *)
-    index: Ligo.tez;
-    protected_index: Ligo.tez;
+    index: Ligo.nat;
```

Comment:
Perhaps we should add a note here that index has units `tez/kit` for clarity.

Reply:
Hmmm, actually, I am not entirely sure that these are the units for the index, now that you mention it. It was `tez` before, the [original doc](https://hackmd.io/teMO2x9PRRy1iTBtrSMBvA#Target-and-quantity) said _"expressed in kit"_, and now you mentioned `tez/kit` :joy: Agreed about figuring out (and adding comments about) the units of measure, but I think it might be best to do that in a separate PR.

Reply:
:thinking: hmm good point. That approach also sounds good to me